rover/usbreset.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+import os
+import fcntl
+import sys
+
+device = sys.argv[1]
+print("resetting driver:" + device)
+USBDEVFS_RESET= 21780
+
+try:
+    with open(device ,'w', os.O_WRONLY) as f:
+        fcntl.ioctl(f, USBDEVFS_RESET, 0)
+        print("Successfully reset usb")
+except Exception as msg:
+    print("Failed to reset device: ", msg)
```

Comment:
why do you have python and c versions of this?

Reply:
The C version wasn't resetting the CAN bus when CAN buffer was full, so I just wanted to try out the python version. It was without luck, of course.